src/PowerShellEditorServices/Services/Extension/ExtensionService.cs

```diff
@@ -140,9 +173,27 @@ public Task InvokeCommandAsync(string commandName, EditorContext editorContext,
                         WriteOutputToHost = !editorCommand.SuppressOutput,
                         AddToHistory = !editorCommand.SuppressOutput,
                         ThrowOnError = false,
-                    });
+                    }).ContinueWith((Task executeTask) =>
+                    {
+                        ExecutionStatus status = ExecutionStatus.Failed;
+                        if (executeTask.IsCompleted)
+                        {
+                            status = ExecutionStatus.Completed;
+                        }
+                        else if (executeTask.IsCanceled)
+                        {
+                            status = ExecutionStatus.Aborted;
+                        }
+                        else if (executeTask.IsFaulted)
+                        {
+                            status = ExecutionStatus.Failed;
```

Comment:
`ThrowOnError` is false do I don't think this branch is reachable.

...that said we currently don't have a way to surface whether something had errors, so 🤷 I'm cool with leaving it until we figure that out 😁

Reply:
TBH I have no idea how that would interact with the `Task.IsFaulted` field.